frontend/src/metabase/app.js

```diff
@@ -70,12 +71,16 @@ function _init(reducers, getRoutes, callback) {
   root.render(
     <Provider store={store}>
       <EmotionCacheProvider>
-        <DragDropContextProvider backend={HTML5Backend} context={{ window }}>
-          <ThemeProvider>
-            <GlobalStyles />
-            <Router history={history}>{routes}</Router>
-          </ThemeProvider>
-        </DragDropContextProvider>
+        <DragDropContextProviderContext.Provider
+          value={DragDropContextProvider}
```

Comment:
Using the `DragDropContextProvider` component as a value here seems off to me.

- I see we consume this context as `isDnDAvailable`, but wouldn't the context provider component always be available?
- Could we pass a boolean as a context value here or have the boolean in the Redux store instead? I see we only need to check if the DnD is available, but we don't need to use the context provider itself.

Reply:
Done, I put it in the `app` slice